doc/Caliptra.md

```diff
@@ -915,7 +915,7 @@ Fields are little endian unless described otherwise.
 | Vendor LMS public key index | 4 | The hint to ROM to indicate which LMS public key it should first use. |
 | Flags | 4 | Feature flags. <br> **Bit0:** - Interpret the pl0_pauser field. If not set, all PAUSERs are PL1 <br>**Bit1-Bit31:** Reserved |
 | TOC Entry Count | 4 | Number of entries in TOC. |
-| PL0 PAUSER | 4 | The PAUSER with PL0 privileges. |
+| PL0 PAUSER | 4 | The PAUSER with PL0 privileges. This value is used by the RT FW to verify the caller privilege against its PAUSER wired through APB. |
```

Comment:
Unclear subject: what is "wired through APB"? If it's the PAUSER, then I recommend the following edit to clarify the subject:

This value is used by the RT FW to verify the caller privilege against its PAUSER. The PAUSER is wired through APB.

Reply:
Reading this again, while it is true that PAUSER is set via APB, but from Caliptra's perspective we are just using the [mailbox user](https://chipsalliance.github.io/caliptra-rtl/main/external-regs/?p=caliptra_top_reg.mbox_csr.mbox_user) register. Maybe it's more clear to just reference that.

Reply:
That register is RO both sides, the integration must have another mechanism to set the PAUSER